test/verify/check-pages

```diff
@@ -1009,6 +1009,30 @@ OnCalendar=daily
         self.assertEqual(str(user_info["id"]), m.execute("id -u admin").strip())
         self.assertEqual(str(user_info["gid"]), m.execute("id -g admin").strip())
 
+    @testlib.skipImage("No SELinux", "debian-*", "ubuntu-*", "arch")
+    def testFSReplaceSELinuxContext(self) -> None:
+        b = self.browser
+        m = self.machine
+
+        path = "/tmp/custom-selinux-context"
+        custom_context = "system_u:object_r:proc_t:s0"
+        m.execute(f"""
+            touch {path}
+            chcon {custom_context} {path}
+        """)
+
+        self.login_and_go("/playground/test")
+        b.wait_visible("#fsreplace1-create")
+        b.set_input_text("#fsreplace1-filename", path)
+        b.set_input_text("#fsreplace1-content", "data")
+        b.set_checked("#fsreplace1-append", val=True)
+        b.click("#fsreplace1-create")
+        b.wait_visible("#fsreplace1-create:not(:disabled)")
+
+        self.assertEqual(m.execute(f"cat {path}"), "data")
+        se_context = m.execute(f"stat --format=%C {path}").strip()
+        self.assertEqual(se_context, custom_context)
```

Comment:
Please add a test without superuser. I figure you can't change the user and subject role then, but the type ought to be changeable.

Reply:
Yup, makes sense and it works \o/